### PR TITLE
FIX #2823: Profile list activity label

### LINKED ISSUE #2823: [A11y] Add label for ProfileListActivity
## Task
Introduce label for `ProfileListActivity` as `Profile List`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by pravesh00**
## Explanation

Fixes #2823: Profile list activity label added

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by pravesh00** [comment]
PTAL @Victor-Titan @rt4914

**Comment by pravesh00** [comment]
@rt4914 can you please help me with how to remove these unit test failures?

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/misc.xml:9]: This changes from this entire file. It should not be visible under `Files changed` section on github.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -435,4 +435,5 @@
   <string name="audio_player_on">Audio, ON</string>
   <string name="audio_player_off">Audio, OFF</string>
   <string name="hints_android_solution_correct_answer">%s/%s</string>
+  <string name="profile_list_activity_label">Profile List</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: Rename to `profile_list_activity_title`

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -121,7 +121,8 @@
     <activity android:name=".app.settings.profile.ProfileEditActivity" />
     <activity
       android:name=".app.settings.profile.ProfileListActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/profile_list_activity_label"/>
```
Comment (rt4914) [inline app/src/main/AndroidManifest.xml]: Add ` ` (space) before `/>`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@pravesh00 Suggestion changes.

Also if you check sample PR number mentioned in the issue that PR contains a test too. You have to create a similar test file and add test. You can create a new test file `ProfileListActivityTest`

**Comment by pravesh00** [comment]
I was facing some gradle change issues so I have closed this PR and created a new one #3244

**Pull request closed without merging**